### PR TITLE
Limit simple search

### PR DESCRIPTION
**Pull request by smithellis**
Limit simple search - don't display archived Questions.

### CODE REVIEW

File: kitsune/search/views.py [hunk 1]
```diff
@@ -100,7 +100,9 @@ def simple_search(request):
 
     # apply aaq/kb configs
     if cleaned["w"] & constants.WHERE_WIKI:
-        search.add(WikiSearch(query=cleaned["q"], locale=language, product=product))
+        search.add(
+            WikiSearch(query=cleaned["q"], locale=language, product=product, is_archived=False)
+        )
     if cleaned["w"] & constants.WHERE_SUPPORT:
         search.add(QuestionSearch(query=cleaned["q"], locale=language, product=product))
```
Comment (akatsoulas) [inline kitsune/search/views.py:107]: Let's do the same here

**Review by escattone — COMMENTED** [review]
@smithellis Have you tried testing this locally? When I run this branch locally, I stop getting any hits when making simple searches for known titles or keywords. I thought it was my setup at first, but when I switched to the `main` branch, my simple searches for known titles and keywords worked finr.

File: kitsune/search/search.py [hunk 2]
```diff
@@ -124,6 +125,8 @@ def get_filter(self):
                     "gte": datetime.now(timezone.utc) - timedelta(days=QUESTION_DAYS_DELTA)
                 },
             ),
+            # don't return archived Questiosn
```
[the remark this answers is not in the document]
Comment (smithellis) [reply kitsune/search/search.py:128]: It's been a while, but yes I did test locally. I'll take another look. And fix the typo, too!

**Review by escattone — APPROVED** [review]
Works great now, thanks @smithellis.

**Pull request merged**